sideboard/jsonrpc.py

```diff
@@ -6,6 +6,7 @@
 from cherrypy.lib.jsontools import json_decode
 
 from sideboard.lib import log, config, serializer
+from sideboard.websockets import trigger_delayed_notifications
```

Comment:
This is the ``delay=True`` change on ``notify`` mentioned above.  Here's the link:
https://github.com/magfest/sideboard/pull/82/files#r115605657